<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartForm</class>
 <widget class="QWidget" name="ChartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Диаграмма</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: сentury gothic;
font-size: 14px;
background-color: rgb(70, 70, 70);
}</string>
  </property>
  <widget class="QPushButton" name="backToMainWindowPushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: red;
border-radius: 60px;
font-family: сentury gothic;
font-size: 24px;
color: white;
border-radius: 15px;
}

QPushButton:hover
{
color: #333;
border-radius: 15px;
background: #49ebff;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {
    border: 1px solid black;
    background: white;
}

QTabWidget::tab-bar:top {
    top: 1px;
}

QTabWidget::tab-bar:bottom {
    bottom: 1px;
}

QTabWidget::tab-bar:left {
    right: 1px;
}

QTabWidget::tab-bar:right {
    left: 1px;
}

QTabBar::tab {
    border: 1px solid black;
}

QTabBar::tab:selected {
    background: white;
}

QTabBar::tab:!selected {
    background: silver;
}

QTabBar::tab:!selected:hover {
    background: #999;
}

QTabBar::tab:top:!selected {
    margin-top: 3px;
}

QTabBar::tab:bottom:!selected {
    margin-bottom: 3px;
}

QTabBar::tab:top, QTabBar::tab:bottom {
    min-width: 8ex;
    margin-right: -1px;
    padding: 5px 10px 5px 10px;
}

QTabBar::tab:top:selected {
    border-bottom-color: none;
}

QTabBar::tab:bottom:selected {
    border-top-color: none;
}

QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one {
    margin-right: 0;
}

QTabBar::tab:left:!selected {
    margin-right: 3px;
}

QTabBar::tab:right:!selected {
    margin-left: 3px;
}

QTabBar::tab:left, QTabBar::tab:right {
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;
}

QTabBar::tab:left:selected {
    border-left-color: none;
}

QTabBar::tab:right:selected {
    border-right-color: none;
}

QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one {
    margin-bottom: 0;
}</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Вопросы</string>
    </attribute>
    <widget class="QChartView" name="chartView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>721</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QComboBox" name="questionNumberComboBox">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>160</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
border: 1px solid #ced4da;
color: rgb(255, 255, 255);
}

QComboBox::drop-down
{
border: 0px;
}

QComboBox::down-arrow
{
image: url(:/images/arrow-down.png);
width: 10px;
height: 10px;
margin-right: 5px;
}

QComboBox::on
{
border: 4px solid #c2dbfe
}

QComboBox QListView
{
font-size: 15px;
	background-color: rgb(255, 255, 255);
}

QComboBox QListView::item
{
	background-color: rgb(255, 255, 255);
}

QComboBox QListView::item:hover
{
background-color: #1e90ff;
}

QComboBox QListView::item:selected
{
background-color: #1e90ff;
}</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9 вопрос</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 вопрос</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Общая диаграмма</string>
    </attribute>
    <widget class="QChartView" name="chartView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>881</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Клиенты</string>
    </attribute>
    <widget class="QChartView" name="chartView_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QComboBox" name="userNameComboBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>170</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
border: 1px solid #ced4da;
color: rgb(255, 255, 255);
}

QComboBox::drop-down
{
border: 0px;
}

QComboBox::down-arrow
{
image: url(:/images/arrow-down.png);
width: 10px;
height: 10px;
margin-right: 5px;
}

QComboBox::on
{
border: 4px solid #c2dbfe
}

QComboBox QListView
{
font-size: 15px;
	background-color: rgb(255, 255, 255);
}

QComboBox QListView::item
{
	background-color: rgb(255, 255, 255);
}

QComboBox QListView::item:hover
{
background-color: #1e90ff;
}

QComboBox QListView::item:selected
{
background-color: #1e90ff;
}</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="companyNameLabel">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>220</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family: сentury gothic;
font-size: 16px;
	color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="fieldOfCompanyLabel">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>250</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family: сentury gothic;
font-size: 16px;
	color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius: 15px;
border: 1px solid rgb(37, 39, 48);
color: #FFF;
text-align: center;
padding-left: 20px;
padding-right: 20px;
font-size: 16px;
}</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="searchLabel">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>Поиск по имени: </string>
     </property>
    </widget>
    <widget class="QToolButton" name="searchToolButton">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{
border: none;
}

QToolButton:hover
{
color: #333;
background: rgb(231, 231, 231);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Компании</string>
    </attribute>
    <widget class="QChartView" name="chartView_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="searchLabel_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Поиск по компании: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLineEdit_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius: 15px;
border: 1px solid rgb(37, 39, 48);
color: #FFF;
text-align: center;
padding-left: 20px;
padding-right: 20px;
font-size: 16px;
}</string>
     </property>
     <property name="maxLength">
      <number>35</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="companyNameComboBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>170</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
border: 1px solid #ced4da;
color: rgb(255, 255, 255);
}

QComboBox::drop-down
{
border: 0px;
}

QComboBox::down-arrow
{
image: url(:/images/arrow-down.png);
width: 10px;
height: 10px;
margin-right: 5px;
}

QComboBox::on
{
border: 4px solid #c2dbfe
}

QComboBox QListView
{
font-size: 15px;
	background-color: rgb(255, 255, 255);
}

QComboBox QListView::item
{
	background-color: rgb(255, 255, 255);
}

QComboBox QListView::item:hover
{
background-color: #1e90ff;
}

QComboBox QListView::item:selected
{
background-color: #1e90ff;
}</string>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="searchToolButton_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{
border: none;
}

QToolButton:hover
{
color: #333;
background: rgb(231, 231, 231);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
